test/lit/wasm-split/symbolmap.wast

```diff
@@ -0,0 +1,24 @@
+;; RUN: wasm-split %s --keep-funcs=bar -o1 %t.1.wasm -o2 %t.2.wasm --symbolmap
```

Comment:
what is `%t` here? a temp filename?

Reply:
Exactly. The full list of available substitutions is here: https://llvm.org/docs/CommandGuide/lit.html#substitutions.